StRoot/StRefMultCorr/StRefMultCorrLinkDef.h

```diff
@@ -0,0 +1,11 @@
+#ifdef __CINT__
+
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+// Main PicoDst classes
+#pragma link C++ class StRefMultCorr+;
+#pragma link C++ class StCentralityMaker+;
```

Comment:
As defined in https://github.com/star-bnl/star-sw/blob/101589368fd7e154f2c263cbe282e276470b3b4c/StRoot/StRefMultCorr/CentralityMaker.h#L91
```suggestion
#pragma link C++ class CentralityMaker+;
```